src/storage/assets/priceApiSlice.ts

```diff
@@ -47,8 +47,14 @@ export const priceApi = createApi({
     }),
     getHistoricalPrice: builder.query<HistoricalPriceResult[], HistoricalPriceQueryParams>({
       query: ({ currency, days }) => `/coins/alephium/market_chart?vs_currency=${currency.toLowerCase()}&days=${days}`,
-      transformResponse: (response: { prices: number[][] }) =>
-        response.prices.map((p) => ({ date: dayjs(p[0]).format(CHART_DATE_FORMAT), price: p[1] }))
+      transformResponse: (response: { prices: number[][] }) => {
+        const { prices } = response
```

Comment:
The very last fetched value has a "lag", which means that  the last 2 `dates` can actually be the same day.

We should probably systematically remove the last value here, and push the latest price in the chart as you've done it.

Reply:
> We should systematically remove the last value here: it makes sense for historical data to stop the day before today IMO. In the chart, we'll add the latest price as you've done in this PR.

That's exactly what's happening here.

What I implemented (and what we actually want to achieve) is to replace the last 2 elements (that seem to always be today's date) with the actual latest price. Since I can't find any documentation that says that the last 2 elements are always going to end up being today's date, I used `filter` instead of `pop`. In the chart component, I then `push` the today's date with the latest price.

Reply:
The last 2 items were yesterday's date during my test, hence the issue I witnessed.

Reply:
You tested today and the last 2 elements were yesterday's date?! Interesting case!